docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Micrometer.adoc

```diff
@@ -9,87 +9,44 @@ ifdef::env-github[]
-(described below). By default, it is `true`. The standalone configurations explicitly sets it to `false` to accept
-unauthenticated access to the HTTP endpoints.
-* `exposed-subsystems` - a list of strings corresponding to the names of subsystems that exposes their metrics in the
-HTTP metrics endpoints. By default, it is not defined (there will be no metrics exposed by subsystem. The special
-wildcard "{asterisk}" can be used to expose metrics from _all_ subsystems. The standalone configuration sets this
-attribute to "{asterisk}".
-
-[[micrometer-metrics-http-endpoint]]
-== HTTP Endpoint
-
-The Micrometer HTTP endpoint is accessible on WildFly HTTP management interface http://localhost:9990/metrics[].
-
-Secured access to the HTTP endpoint is controlled by the `security-enabled` attribute of the `/subsystem=micrometer` resource.
-If it is set to `true`, the HTTP client must be authenticated.
+[IMPORTANT]
+It is assumed that the server administrator will provision and secure the collector, which is outside the scope of this document.
 
-If security is disabled, the HTTP endpoint returns a `200 OK` response:
+Note that this an alternative to the existing WildFly Metrics extension. While they may be run concurrently, it is not advisable, as this will likely have an impact on server performance due to the duplicated metrics collection. To disable WildFly Metrics, issue these commands:
 
+[source,options="nowrap"]
 ----
-$ curl -v http://localhost:9990/metrics
-< HTTP/1.1 200 OK
-...
-# HELP jvm_uptime_seconds Displays the uptime of the Java virtual machine
-# TYPE jvm_uptime_seconds gauge
-jvm_uptime_seconds 215.505
-...
+[standalone@localhost:9990 /] /subsystem=metrics:remove()
+[standalone@localhost:9990 /] /extension=org.wildfly.extension.metrics:remove()
+[standalone@localhost:9990 /] reload
 ----
 
-If security has been enabled, the HTTP client must pass the credentials corresponding to a management user
-created by the `add-user` script. For example:
+== Management Model
 
-----
-$ curl -v --digest -u myadminuser:myadminpassword http://localhost:9990/metrics
-< HTTP/1.1 200 OK
-...
-# HELP jvm_uptime_seconds Displays the uptime of the Java virtual machine
-# TYPE jvm_uptime_seconds gauge
-jvm_uptime_seconds 215.505
-...
-----
+The `/subsystem=micrometer` resource defines two attributes:
 
-If the authentication fails, the server will reply with a `401 NOT AUTHORIZED` response.
+* `endpoint` - the URL of the metrics collector endpoint (default: http://localhost:4318/v1/metrics)
+* `exposed-subsystems` - a list of strings corresponding to the names of subsystems that exposes their metrics in the
+HTTP metrics endpoints. By default, it is not defined (there will be no metrics exposed by subsystem. The special wildcard "{asterisk}" can be used to expose metrics from _all_ subsystems. The standalone configuration sets this attribute to "{asterisk}".
```

Comment:
Minor typo: missing the closing `)` here after `exposed by subsystem`.

Reply:
Fixed